src/js/webflow/import.js

```diff
@@ -9,14 +9,18 @@ exports.transform = function (window, $) {
   $("head").append("<script>dataLayer = [];</script>");
   $("head").append(`
     <script>
-    (function (p, u, s, h, x) { p.pushpad = p.pushpad || function () { (p.pushpad.q = p.pushpad.q || []).push(arguments) }; h = u.getElementsByTagName('head')[0]; x = u.createElement('script'); x.async = 1; x.src = s; h.appendChild(x); })(window, document, 'https://pushpad.xyz/pushpad.js');
-
-    pushpad('init', '7550', {serviceWorkerPath: null});
-    pushpad('widget', {
-      promptTitle: 'Subscribe to Notifications',
-      promptMessage: 'Subscribe to receive instant notifications of service disruptions and other important information.',
-      promptButtonColor: "#0094ff",
-    });
+      const pushpadProjectId = \`${process.env.PUSHPAD_PROJECT_ID}\`;
+      if (!pushpadProjectId !== '') {
```

Comment:
```suggestion
      if (pushpadProjectId !== '') {
```

Reply:
I don't think you want double `not`

Will it be empty string, null or undefined?

Reply:
Thanks for catching that

It'll be an empty string, since it is initialised as a template literal